fearless/Modules/NFT/NftCollection/NftCollectionViewLayout.swift

```diff
@@ -60,13 +72,28 @@ final class NftCollectionViewLayout: UIView {
         navigationBar.snp.makeConstraints { make in
             make.leading.top.trailing.equalToSuperview()
         }
+        imageView.snp.makeConstraints { make in
+            make.top.equalTo(navigationBar.snp.bottom)
+            make.leading.equalToSuperview().offset(UIConstants.defaultOffset)
+            make.trailing.equalToSuperview().inset(UIConstants.defaultOffset)
+            make.height.equalTo(imageView.snp.width)
+        }
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(imageView.snp.bottom)
+            make.leading.trailing.equalTo(imageView)
+        }
         collectionView.snp.makeConstraints { make in
             make.leading.trailing.bottom.equalToSuperview()
-            make.top.equalTo(navigationBar.snp.bottom)
+            make.top.equalTo(titleLabel.snp.bottom)
         }
     }
 
     func bind(viewModel: NftCollectionViewModel) {
         navigationTitleLabel.text = viewModel.collectionName
+        viewModel.collectionImage?.loadImage(on: imageView, targetSize: CGSize(
+            width: UIScreen.main.bounds.width - 16,
```

Comment:
it's incorrect to use magic number here because it depends on leading and trailing insets of imageView. so please use local Constants enum